release.config.js

```diff
@@ -0,0 +1,24 @@
+export default {
+  branches: [
+    'main',
+  ],
+  plugins: [
+    ['@semantic-release/commit-analyzer', { preset: 'conventionalcommits' }],
+    '@semantic-release/release-notes-generator',
+    '@semantic-release/changelog',
+    '@semantic-release/npm',
+    [
+      '@semantic-release/git',
+      {
+        assets: [
+          'package.json',
+          'package-lock.json',
+          'CHANGELOG.md',
+        ],
+        // eslint-disable-next-line no-template-curly-in-string
+        message: 'chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}',
+      },
+    ],
+    '@semantic-release/github',
+  ],
```

Comment:
```suggestion
    '@semantic-release/commit-analyzer',
    '@semantic-release/release-notes-generator',
    '@semantic-release/changelog',
    '@semantic-release/npm',
    [
      '@semantic-release/git',
      {
        assets: [
          'package.json',
          'package-lock.json',
          'CHANGELOG.md',
        ],
        // eslint-disable-next-line no-template-curly-in-string
        message: 'chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}',
      },
    ],
    '@semantic-release/github',
  ],
  preset: 'conventionalcommits',
```
[Upon further reading,](https://github.com/semantic-release/semantic-release/blob/HEAD/docs/usage/plugins.md#plugin-options-configuration) `commit-analyzer`, `release-notes-generator`, and `changelog` all need to see this preset option, which we can define across the board by moving it's key to the same level as `plugins`.

Reply:
There is also mention of a key called `presetConfig` which semantic release _says_ we need to define if we're using `conventionalcommits` as a preset, but the [preset has defaults built in?](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-conventionalcommits/src/index.js) Not sure how to test whether we really need this set or not.

Reply:
I tested what we have now in a dry run mode - it detected a breaking change on `feat!` so we're good